Sources/ClientRuntime/Errors/DefaultError.swift

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0.
+ */
+
+import Foundation
+import ClientRuntime
+
+/// A general Error Structure for Rest JSON protocol
```

Comment:
What is this for?

Reply:
A error struct used by the test generator, will move it to `SmithyTestUtil` swift package instead of `ClientRuntime`